alpaca_trade_api/polygon/streamconn.py

```diff
@@ -190,12 +190,15 @@ def run(self, initial_channels=[]):
 
     async def close(self):
         '''Close any open connections'''
-        if self._consume_task:
-            self._consume_task.cancel()
+        await self.cancel_task()
         if self._ws is not None:
             await self._ws.close()
         self._ws = None
 
+    async def cancel_task(self):
```

Comment:
[minor] should be private: `_cancel_task()`

Reply:
it is used outside of the class so it shouldn't be private